packages/extension-bubble-menu/src/bubble-menu-plugin.ts

```diff
@@ -25,21 +26,29 @@ export class BubbleMenuView {
 
   public view: EditorView
 
+  public hideWhenSelecting: boolean
+
   public preventHide = false
 
+  public viewSelecting = false
+
   public tippy!: Instance
 
   constructor({
     editor,
     element,
+    hideWhenSelecting = false,
     view,
     tippyOptions,
   }: BubbleMenuViewProps) {
     this.editor = editor
     this.element = element
+    this.hideWhenSelecting = hideWhenSelecting
     this.view = view
     this.element.addEventListener('mousedown', this.mousedownHandler, { capture: true })
     this.view.dom.addEventListener('dragstart', this.dragstartHandler)
+    this.view.dom.addEventListener('mousedown', this.viewMousedownHandler)
+    this.view.dom.addEventListener('mouseup', this.viewMouseupHandler)
```

Comment:
We wanted the bubble menu to also hide while a selection was in progress with the keyboard. This might be lower priority, as the menu doesn't get in the way of the selection.

If you mouse up outside of the view (or window), the selection should also be considered done. A pattern for this is to **not** add the `mouseup` listener here....